### PR TITLE
fix: Improve og images

### PR DESCRIPTION
**Pull request by soniaklimas**
I want to merge this change because it adjust opengraph images.

# Pull Request Checklist

- [ ] Test the changes locally to ensure they work as expected.
- [ ] Document the testing process and results in the pull request description. (Screen recording, screenshot etc)
- [ ] Include new tests for any new functionality or significant changes.
- [ ] Ensure that tests cover edge cases and potential failure points.
- [ ] Document the impact of the changes on the system, including potential risks and benefits.
- [ ] Provide a rollback plan in case the changes introduce critical issues.
- [ ] Update documentation to reflect any changes in functionality.

### CODE REVIEW

File: apps/storefront/src/app/[locale]/(main)/products/[slug]/opengraph-image.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
+/* eslint-disable @next/next/no-img-element */
```
Comment (karolkarolka) [inline apps/storefront/src/app/[locale]/(main)/products/[slug]/opengraph-image.tsx:1]: why do you need here eslint disable?

Comment (soniaklimas) [reply apps/storefront/src/app/[locale]/(main)/products/[slug]/opengraph-image.tsx:1]: Because I use  tag and it causes warnings

Comment (karolkarolka) [reply apps/storefront/src/app/[locale]/(main)/products/[slug]/opengraph-image.tsx:1]: can you disable the error causing line instead of the whole file?

**Review by karolkarolka — APPROVED** [review]
